micro-rdk/src/common/ota.rs

```diff
@@ -103,20 +103,37 @@ struct EspAppDesc {
     reserv2: [u32; 20],
 }
 
+#[derive(Error, Debug)]
+pub(crate) enum ConfigError {
+    #[error("version {0} has length {1}, maximum allowed characters is {2}")]
+    InvalidVersionLen(String, usize, usize),
+    #[error("the given url {0}, is not a valid URI: {1}")]
```

Comment:
- Is the url vs uri distinction in this message intentional and making a subtle point, or should it be one or the other?
- If one is capitalized, I'd expect the other to be, or vice versa.

Reply:
it was making a subtle point, but I've decided to go with URL (lowercase) to be consistent. The underlying error may mention it regardless